README.md

```diff
@@ -7,8 +7,8 @@
 - Manage storage & databases with a unified Python API ("lakehouse").
 - Track data lineage across notebooks & pipelines.
 - Integrate registries for experimental metadata & in-house ontologies.
-- Validate, standardize & annotate.
-- Collaborate across a mesh of distributed LaminDB instances.
```

Comment:
See my note below. "mesh" is a well-known keyword in the field that triggers a certain connotation for the most important reader audience.

But given this is a simplification and maintains "distributed", I think it's OK to risk not triggering the connotation. People who understand mesh should also understand that it's implied by "distributed".

Reply:
Thanks for spotting this!